Sources/Impl/PinLayoutImpl.swift

```diff
@@ -234,20 +234,63 @@ class PinLayoutImpl: PinLayout {
         left({ "all() left coordinate" })
         return self
     }
+
+    @discardableResult
+    func all(_ value: CGFloat) -> PinLayout {
+        func context() -> String { return "all(\(value))" }
+        top(value, context)
+        bottom(value, context)
+        left(value, context)
+        right(value, context)
+        return self
+    }
     
     @discardableResult
     func horizontally() -> PinLayout {
         right({ "horizontally() right coordinate" })
         left({ "horizontally() left coordinate" })
         return self
     }
+
+    @discardableResult
+    func horizontally(_ value: CGFloat) -> PinLayout {
+        func context() -> String { return "horizontally(\(value))" }
+        left(value, context)
+        right(value, context)
+        return self
+    }
+
+    @discardableResult
+    func horizontally(_ percent: Percent) -> PinLayout {
+        func context() -> String { return "horizontally(\(percent.description))" }
+        left(percent, context)
+        right(percent, context)
+        return self
+    }
+
     @discardableResult
     func vertically() -> PinLayout {
         top({ "vertically() top coordinate" })
         bottom({ "vertically() bottom coordinate" })
         return self
     }
 
+    @discardableResult
+    func vertically(_ value: CGFloat) -> PinLayout {
+        func context() -> String { return "vertically(\(value))" }
+        top(value, context)
```

Comment:
Could you do as in `all()`, i.e. set a different context string for all internal calls (ex: `{ "vertically(\(value)) top coordinate" }`)